static-analysis/teammates-eslint-es6.yml

```diff
@@ -20,6 +20,15 @@ rules:
     # enforce a maximum line length
     max-len: ["error", 125]
 
+    # disallow continue statements
+    no-continue: "off"
+
+    # disallow mixes of different operators
+    no-mixed-operators: "off"
```

Comment:
`no-continue`: are there good reasons to completely forbid `continue`? Consider:
```javascript
if (condition) {
    continue;
}
doSomething();
```
vs
```javascript
if (condition) {
    doSomething();
}
```
The second variant is prone to creating arrowhead code.